services.json

```diff
@@ -381,10 +387,7 @@
     "addPlugins": [
       "Aws::EC2::Plugins::CopyEncryptedSnapshot",
       "Aws::EC2::Plugins::RegionValidation"
-    ],
-    "dependencies": {
-      "aws-sigv4": "~> 1.0"
```

Comment:
I'm thinking that we are getting rid of the `aws-sigv4` dependency here since it is already noted in the core, right? No reason to restate this dependency in both areas and etc.

Reply:
Not only is it a dependency in core - the codegen already separately adds it as a dependency: https://github.com/aws/aws-sdk-ruby/blob/version-3/build_tools/services.rb#L162